corehq/apps/ota/utils.py

```diff
@@ -165,3 +168,21 @@ def get_restore_user(domain, couch_user, as_user):
         restore_user = couch_user.to_ota_restore_user(domain)
 
     return restore_user
+
+
+def handle_401_response(f):
```

Comment:
I think it would we worth a comment/doc string explaining what the purpose of this is.